include/openenclave/host_verify.h

```diff
@@ -122,6 +122,68 @@ oe_result_t oe_verify_attestation_certificate_with_evidence(
     oe_verify_claims_callback_t claim_verify_callback,
     void* arg);
 
+/**
```

Comment:
I don't like the idea of having duplicate definitions in different files.
Can we have a header file organization that reuses a common header that is included by both host_verify.h and attestation/verifier.h that includes everything that's identical in both?

Reply:
Thanks for highlighting the duplication. openenclave/attestation/verifier.h is included in oehostverify. Removed the functions  oe_verify_evidenece() and oe_verify_attestation_certificate_with_evidence() from host_verify.h